README.md

```diff
@@ -1,176 +1,101 @@
 
-## Specific modifications
-The following modifications are needed before starting to work on the micro-service:
-* In the **gradle.properties** modify the version, projectName, packageLastPartName (the last part in package name).<br>
-   The version should correspond to current release snaphot. This version will be used for downloading the specific version of libraries for dependency management.
-* In the **src/main/resources/log4j2.xml** change the log file name from 'micro-service.log'
-* In the **src/main/webapp/WEB-INF/applicationContext.xml** specify the correct package name in base-package (change the microservice_template name to the packageLastPartName already specified in properties)
-* In the main/java modify the **package name** to your need.
-These changes will impact at the springBoot section in build.gradle file. By default the Application.java from this package will start on bootRun.
+## Installation
 
-## Prepare your IDE
-In the project is used Lombok library. It is needed to install plugin to be able to use it.
-In the IntelliJ IDEA to be able to see Lombok generated code, you should enable the annotation preprocessor. [See documentation here.](https://www.jetbrains.com/help/idea/2016.1/configuring-annotation-processing.html) (Settings->Build->Compiler->Annotation Processors: Enable annotation preprocessing)
+To use SAL as a microservice, follow these steps:
 
-## Building and running the micro-service
+1. Clone the repository:
 
-You can start a microservice as a standalone application:
-```
-$ gradlew clean build bootRun
+```bash
+git clone https://github.com/ow2-proactive/scheduling-abstraction-layer.git
+cd scheduling-abstraction-layer
 ```
 
-You can build a WAR file as follows:
+2. Build the microservice:
 
-```
-$ gradlew clean build war
+```bash
+./gradlew clean build
 ```
 
-Then, you can directly deploy the service with embedded Tomcat:
+To use SAL as a Docker container, pull the public Docker image from DockerHub:
 
-```
-$ java -jar build/libs/microservice-template-X.Y.Z-SNAPSHOT.war
+```bash
+docker pull activeeon/sal
 ```
 
-The WAR file produced by Gradle can also be deployed in the embedded Jetty container started by an instance of [ProActive Server](https://github.com/ow2-proactive/scheduling).
+## Usage
 
-Sometimes the gradle processes are not killing properly when you stop the running application. If you receive the message "the port is already in use" on starting microservice, then kill all suspending gradle processes for previous task. You can do it manually or use in IntelliJ IDEA Gradle killer plugin.
+SAL can be used either as a standalone microservice or as a Docker container. Choose the approach that best suits your requirements.
```

Comment:
How about having this just under ## Installation ?
First you specify how SAL can be used, and after you detail the process.
Your call.

Reply:
I agree with you. i will refactor this.